.github/workflows/weekly.yaml

```diff
@@ -2,13 +2,9 @@ name: Weekly Tests
 
 on:
   schedule:
-    - cron: "0 6 * * *"
+    - cron: "0 6 * * 0"
   push:
     branches: [main]
-  workflow_dispatch:
```

Comment:
Do you want to keep this, so you can trigger the workflow manually?

Reply:
I thought it might be handy for testing/debugging purposes.

Reply:
The `workflow_dispatch:` line enables the manual trigger , see [doco](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)

Reply:
Fixed.